modules/ROOT/pages/install-aws.adoc

```diff
@@ -138,18 +137,25 @@ Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
 
 
 [%header,cols="3*a"]
-.Optional Environment Variables
 |===
 |Variable | Description | Example
+| `controllers` | The number of controller machines to provision. The default value is `3`. | `5`
+| `disable_selinux` | A boolean variable that enables/disables SELinux on the host operating system. The default is `false`. | `true`
```

Comment:
slight reword: "A boolean variable that disables SELinux.."
It comes enabled out of the box on certain operating systems, we only disable it when this var is passed.